src/commands/admin/grantrole.ts

```diff
@@ -0,0 +1,37 @@
+import { Arg } from "@oclif/core/lib/interfaces";
+import { ethers } from "ethers";
+import { TransactionCommand } from "../../base";
+import { getSigner, parseAddress, pretty, run } from "../../helpers";
+
+export default class GrantRole extends TransactionCommand {
+  static summary = "Grant a role to an account on the Armada Network.";
```

Comment:
can you also add a summary for the admin topic here https://github.com/armada-network/armada-cli/blob/main/package.json#L40. something like "Perform admin functions such as grantRole"